impl-base/src/main/java/org/jboss/shrinkwrap/impl/base/URLPackageScanner.java

```diff
@@ -85,15 +90,12 @@ public static URLPackageScanner newInstance(boolean addRecursively, final ClassL
     /**
      * Factory method to create an instance of URLPackageScanner in the default package
      *
-     * @param pkg
-     *            Package that will be scanned
-     * @param addRecursively
-     *            flag to add child packages
-     * @param classLoader
-     *            class loader that will have classes added
+     * @param addRecursively flag to add child packages
+     * @param classLoader    Class loader that will have classes added
+     * @param callback       Callback to invoke when a matching class is found
      * @return new instance of URLPackageScanner
      */
-    public static URLPackageScanner newInstance(boolean addRecursively, ClassLoader classLoader, Callback callback) {
+    public static URLPackageScanner newInstance(boolean addRecursively, ClassLoader classLoader, Callback callback) throws IOException {
```

Comment:
throws IOE, but we catch it and swallow?  Need to throw here in the method declar?  Would prefer not if we're just going to swallow it or wrap in RuntimeException.